docker/test/fuzzer/query-fuzzer-tweaks-users.xml

```diff
@@ -11,6 +11,10 @@
                     <max>10</max>
                 </max_execution_time>
 
+                <max_memory_usage>
+                    <max>10GB</max>
```

Comment:
```suggestion
                    <max>10737418240</max>
```

Reply:
@qoega `10GB` is expected to work, Misha Korotov added this feature one year ago.